src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -313,6 +313,13 @@ def get_api_version(self, resource_type=None, operation_group=None):
         resource_type = resource_type or self.command_kwargs.get('resource_type', None)
         return self.loader.get_api_version(resource_type=resource_type, operation_group=operation_group)
 
+    def get_param_min_api_version(self, parameter, parent_locals):
```

Comment:
just a wild thinking, do you think it make sense that cli might be asked to have multiple min_api_version, such as one for azure stack, one for JEDI, one for moon_cake?

Reply:
Currently, min_api is based on the first time this param appears in python SDK/swagger spec. Unless we need to choose different package version(not api-version) of sdk for different cloud, I cannot image we need to support multiple min_api_version.

Reply:
Since min API version is constant (will not change in the future), I prefer hard-code it to eliminate the over-complexity.

Reply:
it's not about one api version, pls look at the whole file, or cli codes in general, lots of api version, even in one rp, one resource, min_api_version is on property level. This introduces bad readability and heavy maintenance effort. purpose of the comments is general improving code readability, maintenance. while for how to generalize the way, we can discuss. that's the context why @MyronFanQiu  provided 2 options open for discussion. If more concern, let's discuss offline.